Index: TransportationManagement/models.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- TransportationManagement/models.py	(date 1543161354085)
+++ TransportationManagement/models.py	(date 1543161354085)
@@ -73,7 +73,7 @@
     ZSCNo = models.ForeignKey(to=Car, on_delete=None)
     SGCNo = models.CharField(max_length=5)
     ZSDNo = models.ForeignKey(to=Driver, on_delete=None)
-    ZSDName = models.ForeignKey(to=Driver, to_field=Driver.DName, on_delete=None)
+    ZSDName = models.ForeignKey(to=Driver,  on_delete=None)
     Time = models.DateTimeField()
     Spot = models.CharField(max_length=50)
     Cause = models.TextField()
